chainer/training/extensions/_snapshot.py

```diff
@@ -4,7 +4,7 @@
 from chainer.utils import argument
 
 
-def snapshot_object(target, filename, savefun=npz.save_npz, **kwargs):
+def snapshot_object(target, filename, **kwargs):
```

Comment:
To make the documentation more readable, could you change the docstring as follows?

```python
def snapshot_object(target, filename, **kwargs):
    """snapshot_object(target, filename, *, savefun=npz.save_npz, \
condition=None, writer=None, snapshot_on_error=False)

    Returns a trainer extension to take snapshots of a given object.

    ...
```

Reply:
Done